This GitHub contains a list of ressources for a better understanding of the single-cell genomic field. These ressources obviously include landmark papers, but also blogs, YouTube videos and channels.


## Websites, videos and blogs

- **Likely one of the most important and valuable ressource in the field**: the list and technical details of every published single-cell protocol aggregated by the Teichman lab available [here](https://teichlab.github.io/scg_lib_structs/).
- The list 
- The excellent blog of Valentine Svensson that focuses on the computational analysis of single-cell data: [What do you mean 'heterogeneity'](https://www.nxn.se/)
-  An in-depth description of the different dimensionality reduction methods commonly used in the single-cell genomic field such as t-SNE and UMAP. Available in the [smallvis GitHub](https://jlmelville.github.io/smallvis/). More broadly this GitHub contains a large number of implementations and other valuable ressources.
-  The Broad institute provides a list of very interesting computational biology YouTube videos, including videos dedicated to single-cell data analysis. More than a hundred videos are available [here](https://www.youtube.com/playlist?list=PLlMMtlgw6qNjROoMNTBQjAcdx53kV50cS).


## Landmark papers

### The prehistory of single-cell analysis


- If you read spanish: **Estructura de los centros nerviosos de las aves**. Of course the interest is more historical than scientific
- One of the first example of biological sytem **The biochemical basis of an all-or-none cell fate switch in Xenopus oocytes**
- First paper prooving that cancer cells can switch between various states: **Stochastic State Transitions Give Rise to Phenotypic Equilibrium in Populations of Cancer Cells**
- Analysis of the NF-kB oscillations in individual cells: **Single-cell NF-κB dynamics reveal digital activation and analog information processing in cells**.
- f



### How does single-cell genomic work ?

- Optimisation of RNA-seq protocols to reach single-cell resolution : **Full-length mRNA-Seq from single-cell levels of RNA and individual circulating tumor cells**.
- The first large-scale plate-based scRNA-seq protocol (MARS-seq): **Massively Parallel Single-Cell RNA-Seq for Marker-Free Decomposition of Tissues into Cell Types**. Its second version is published a few years later: **MARS-seq2.0: an experimental and analytical pipeline for indexed sorting combined with single-cell RNA sequencing**.
- One of the first droplet-based protocol applied to genomic (Drop-Seq):  **High-Throughput Droplet Digital PCR System for Absolute Quantitation of DNA Copy Number**
- The paper introducing Unique Molecular Identifier: **Counting absolute numbers of molecules using unique molecular identifiers**
- First droplet-based scRNA-seq protocol: **Highly Parallel Genome-wide Expression Profiling of Individual Cells Using Nanoliter Droplets**.
- The paper introducing the 10X Chromium® platform: **Massively parallel digital transcriptional profiling of single cells**.
- A nice summary of the different scRNA-seq techniques: **Exponential scaling of single-cell RNA-seq in the past decade**. The paper is of course outdated but is a nice summary of the different technologies available at the time.
- Paper introducing CUT&Tag for single-cell epigenomic: **CUT&Tag for efficient epigenomic profiling of small samples and single cells**

### How to analyze single-cell data ?

- If there is only one paper to read about single-cell data analysis it is the excellent **The triumphs and limitations of computational methods for scRNA-seq** from Peter Kharchenko.
- First paper introducing graph-based clustering under the name of "Phenograph": **Data-Driven Phenotypic Dissection of AML Reveals Progenitor-like Cells that Correlate with Prognosis**.
- A simple yet essential paper from Valentine Svensson: **Droplet scRNA-seq is not zero-inflated**.
- The paper introducing the Velocity approach: **RNA velocity of single cells**.
- The first paper introducing the concept of MetaCells by the Tanay Lab **MetaCell: analysis of single-cell RNA-seq data using K-nn graph partitions**. This approach has been implemented and further optimised by the Pe'er lab in **SEACells infers transcriptional and epigenomic cellular states from single-cell genomics data**.
- f



